File: 01_ProgrammingBasics/_01_FirstStepsInProgrammingExercise/_04_VacationBooksList.java
package _01_ProgrammingBasics._01_FirstStepsInProgrammingExercise;

import java.util.Scanner;

public class _04_VacationBooksList {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input data
        int numberOfPages = Integer.parseInt(scan.nextLine());
        int pagesReadPerHour = Integer.parseInt(scan.nextLine());
        int daysOfReading = Integer.parseInt(scan.nextLine());

        // Calculate hours needed for reading
        int hoursNeededForReading = numberOfPages / pagesReadPerHour;

        // Calculate hours of reading per day
        int hoursPerDay = hoursNeededForReading / daysOfReading;

        // Print the hours of reading per day
        System.out.println(hoursPerDay);
        
        scan.close();
    }
}

File: 23_C#Basics/_01_FirstStepsInProgrammingExercise/_04_VacationBooksList/_04_VacationBooksList.cs
using System;

public class _04_VacationBooksList
{
    public static void Main(string[] args)
    {
        // Read input data
        int numberOfPages = int.Parse(Console.ReadLine());
        int pagesReadPerHour = int.Parse(Console.ReadLine());
        int daysOfReading = int.Parse(Console.ReadLine());

        // Calculate hours needed for reading
        int hoursNeededForReading = numberOfPages / pagesReadPerHour;

        // Calculate hours of reading per day
        int hoursPerDay = hoursNeededForReading / daysOfReading;

        // Print the hours of reading per day
        Console.WriteLine(hoursPerDay);
    }
}
